web/frontend/src/pages/form/components/utils/useQuestionForm.ts

```diff
@@ -142,9 +142,7 @@ const useQuestionForm = (initState: RankQuestion | SelectQuestion | TextQuestion
         const newChoicesMapDefault = new Map(
           Object.entries({
             ...obj,
-            ['en']: obj.en.map((item: string, idx: number) =>
-              idx === index ? e.target.value : item
-            ),
+            en: obj.en.map((item: string, idx: number) => (idx === index ? e.target.value : item)),
```

Comment:
Very subjective but I think it was more readable before 😅 
Furthermore, we try to (roughly) keep a 80 char per line limit.
I haven't checked, but I suspect this is longer than that.

Reply:
Following eslint warning. The `,` is on position 99, so it should just fit :)

What I don't understand is that eslint's [max-len](https://eslint.org/docs/latest/rules/max-len) should cut it off at 80 - should I add a `max-len` entry to `web/frontend/.eslintrc`?